### ISSUE TITLE
Automatic install as a dependency fails

### ISSUE THREAD

**Issue by JJ** [2shortplanks/Test-utf8#4]
Because inc::Module::Install is not previously installed. You need to install it first, but I can't see how you can do that within the current makefile.PL
Anyway, here's what a direct installation with `cpanm` yields:
```
Configuring Test-utf8-1.01
Running Makefile.PL
Can't locate inc/Module/Install.pm in @INC (you may need to install the inc::Module::Install module) (@INC contains: /home/jmerelo/perl5/perlbrew/perls/perl-5.26.1/lib/site_perl/5.26.1/x86_64-linux /home/jmerelo/perl5/perlbrew/perls/perl-5.26.1/lib/site_perl/5.26.1 /home/jmerelo/perl5/perlbrew/perls/perl-5.26.1/lib/5.26.1/x86_64-linux /home/jmerelo/perl5/perlbrew/perls/perl-5.26.1/lib/5.26.1) at Makefile.PL line 1.
BEGIN failed--compilation aborted at Makefile.PL line 1.
```

**Comment by 2shortplanks** [comment]
Fixed by #5

**Comment by cjfields** [comment]
@2shortplanks is this on CPAN yet?   I'm still seeing this issue when installing our tool chain.
